highlight/highlight
# Replace antd toasts

## Summary
Swap out the antd toast element with a Highlight toast element

https://www.loom.com/share/1fc13dede6ff45909ec06ac4c2c46be2?sid=47010324-a73e-4222-a612-1cadaa3f936c

## How did you test this change?
1) View the alerts page
2) Toggle an alert on and off
- [ ] Toast is displayed appropriately
3) View a session on the sessions page
4) Try to download a session
- [ ] First alert displayed appropriately
- [ ] Second alert displayed appropriately (if not enterprise)

## Are there any deployment considerations?
N/A

## Does this work require review from our design team?
@julian-highlight
